docs/UserGuide.md

```diff
@@ -150,8 +165,9 @@ Format: `find person KEYWORD [MORE_KEYWORDS]`
 
 Examples:
 * `find event Hiking` returns `Hiking` and `Park Hiking`
-* `find event Hair Oscars` returns `Hair Cut`, `Oscars`<br>
-  ![result for 'find hair oscars'](images/findHairOscarsResult.png)
+* `find event Hair Oscars` returns `Hair Cut`, `Oscars`
+
+<br>![result for 'find hair oscars'](images/findHairOscarsResult.png)
```

Comment:
Do you think it will be better to remove the <br> as there is already a empty line between the bullet and image?

Reply:
You are right thanks for the suggestion!